### DISCUSSION TITLE
How Should I upload my Documents to apply?

### DISCUSSION THREAD

**Discussion by chishxd** [community/community#143932] [GitHub Education] [answered]
### Hi everyone,

So, as I want to apply for this students pack to get some benefits as a teen with Financial difficulties. My college haven't provided us with any kind of student mail as well. And there's no Date of Admission and other things as stated in GitHub's Documentation, so that image of Student ID got rejected. Though I have an enrollment letter which was given to me last year when I got admitted to this college. But as it is a year old, I think it won't be enough to satisfy the conditions in the policies, and as far as I know, we aren't given any kind of Enrollment letter when passing to second year, but I do have some other proofs like Fee receipt and The form of my scholarship approval this year. So, what should I upload to apply for this plan and get benefit's of GitHub's Students Pack? Thanks for reading, And hope to get some good guidance!

**Comment by huddmax** [comment, marked as answer]
Hi! 👋 

It sounds like you’re in a bit of a tricky situation. Here’s what you can try:

1. **Use the Enrollment Letter**: Even though it’s a year old, GitHub sometimes accepts enrollment letters as long as they clearly show your student status and college details. Try submitting it along with an explanation that it's the most recent official document from your college.

2. **Combine Documents**: Since you also have a fee receipt and a scholarship approval form, consider submitting these additional documents as supporting evidence. This can help show that you’re currently enrolled, even if your college doesn’t issue updated letters each year.

3. **Contact GitHub Education Support**: If these documents don’t work, reach out to [GitHub Education Support](https://support.github.com/contact/education) and explain your situation. They may offer additional guidance or make an exception given your unique case.

Good luck! Hopefully, these documents work out for you, and you get access to the pack soon! 😊

**Reply by chishxd** [reply]
Thanks for the help! Should I upload multiple documents by collaging them into 1 image? Because have them in PDF format or should I get raw images from college?

**Comment by huddmax** [comment]
I don't think they accept PDF format, but send an image of your enrollment certificate. It's also a good idea to confirm your institutional email from the college.

**Reply by chishxd** [reply]
Thanks for your help! I got approval!

**Comment by diyorbekw** [comment]
Hello
